src/xenia/kernel/xam/content_manager.cc

```diff
@@ -252,6 +254,25 @@ X_RESULT ContentManager::DeleteContent(const XCONTENT_DATA& data) {
   }
 }
 
+std::wstring ContentManager::ResolveGameUserContentPath() {
+  wchar_t title_id[9] = L"00000000";
+  std::swprintf(title_id, 9, L"%.8X", kernel_state_->title_id());
+
+  std::string user_name = kernel_state_->user_profile()->name();
+  size_t len = user_name.length();
+  auto user_name_w =
+      reinterpret_cast<wchar_t*>(alloca(sizeof(wchar_t) * (len + 1)));
+  std::mbstowcs(user_name_w, user_name.c_str(), len + 1);
```

Comment:
Use `xe::to_wstring`

Reply:
Thanks.  I've made the change and squashed the changes into one commit.